src/api.js

```diff
@@ -48,6 +48,13 @@ export const createNewUser = async user => {
   }
 };
 
+export const getSearchResults = async input => {
+  const response = await axios.get(
+    `http://localhost:8080/users/search?query=${input}`
```

Comment:
I don't think you need the search bit. You want the end point to be exactly the same as the normal GET list endpoint, but with an extra querystring to filter it down

Reply:
Again, that would be a back end change really - so depends if they approve the back end or not aha

Reply:
Yeah this has been commented in the BE PR as something to think about but not neccesarily something that needs to be done at the moment. Karam has decided that if we do change this it will probably be after the feature freeze.